stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -13,20 +14,23 @@ class AssignManualQuants(models.TransientModel):
     @api.multi
     @api.constrains('quants_lines')
     def check_qty(self):
-        for record in self:
-            if record.quants_lines:
-                move = self.env['stock.move'].browse(
-                    self.env.context['active_id'])
-                if record.lines_qty > move.product_uom_qty:
-                    raise exceptions.Warning(
-                        _('Quantity is higher than the needed one'))
+        precision_digits = dp.get_precision('Product Unit of Measure'
+                                            )(self.env.cr)[1]
```

Comment:
hi, I am not sure about these lines of code, but it is from the bug fixing commit: https://github.com/OCA/stock-logistics-warehouse/commit/173d609ef13f8395dcc8d86858ddac08f66cb639#diff-ee27ef3cc2995f546fa2df998bb3d9b2